src/pages/Location.tsx

```diff
@@ -653,7 +653,7 @@ const Location = () => {
       {awaitingLabwares.length > 0 && (
         <PromptOnLeave
           when={awaitingStorageCheckOnExit}
-          message={'You have labwares that are not stored. Are you sure you want to leave?'}
+          message={'You have labware that are not stored. Are you sure you want to leave?'}
```

Comment:
Please either "labware that is unstored" or preferably "unstored labware"

Reply:
Is this ok - "You have unstored labware. Are you sure you want to leave?"

Reply:
👍